superset/models/user_attributes.py

```diff
@@ -15,8 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 from flask_appbuilder import Model
-from sqlalchemy import Column, ForeignKey, Integer
-from sqlalchemy.orm import relationship
+from sqlalchemy import Column, ForeignKey, Integer  # pylint: disable=wrong-import-order
```

Comment:
Do you know why this error is being thrown? It seems like it would be best to remedy the issue rather than disable the `pylint` error if possible.

Reply:
Oh yeah, not sure what this was about, wrote the PR months ago. I ended up backtracking on `pylint` since 2.x+ is much slower for some reason, removing these edits...